src/EditorFeatures/Core/Extensibility/NavigationBar/INavigationBarPresenter.cs

```diff
@@ -21,8 +21,10 @@ internal interface INavigationBarPresenter
 
         ITextView TryGetCurrentView();
 
-        event EventHandler<EventArgs> ViewFocused;
-        event EventHandler<CaretPositionChangedEventArgs> CaretMoved;
+        /// <summary>
+        /// An event raised when the caret is moved or the active view in the document changed. In either case, the symbol under the caret has changed, so we should refresh.
+        /// </summary>
+        event EventHandler<EventArgs> CaretMovedOrActiveViewChanged;
```

Comment:
@davidwengier We don't implement this interface in VS for Mac, right?

Reply:
No hits for `INavigationBarPresenter` in VS Mac

Reply:
@davidwengier Thanks for confirming!